libs/test-setup/src/lib.rs

```diff
@@ -40,7 +40,6 @@ impl TestAPIArgs {
     }
 }
 
-/// DANGER. This will be used for destructive filesystem access, be careful when changing this. DANGER.
```

Comment:
Not dangerous anymore?

Reply:
Nope, the tests that did filesystem manipulations were deleted.